python/cudf/cudf/io/avro.py

```diff
@@ -16,6 +16,12 @@ def read_avro(
 
     from cudf import DataFrame
 
+    is_single_filepath_or_buffer = ioutils.ensure_single_filepath_or_buffer(
+        path_or_data=filepath_or_buffer, **kwargs,
+    )
+    if not is_single_filepath_or_buffer:
+        raise IOError("filepath_or_buffer resolved to multiple files")
```

Comment:
Not sure `IOError` is the right one. Perhaps ``NotImplementedError` or `ValueError`?